src/components/common/ColonyActions/ActionDetailsPage/DefaultMotion/MotionPhaseWidget/StakingWidget/StakingInput/StakingControls/StakeButton.tsx

```diff
@@ -6,16 +6,28 @@ import { useStakingWidgetContext } from '../../StakingWidgetProvider';
 const displayName =
   'common.ColonyActions.ActionDetailsPage.DefaultMotion.StakingWidget.StakeButton';
 
-const StakeButton = () => {
-  const { canBeStaked, isObjection } = useStakingWidgetContext();
+interface StakeButtonProps {
+  isLoadingData: boolean;
+  enoughTokensToStakeMinimum: boolean;
+  remainingToStake: string;
+}
+
+const StakeButton = ({
+  isLoadingData,
+  enoughTokensToStakeMinimum,
+  remainingToStake,
+}: StakeButtonProps) => {
+  const { isObjection } = useStakingWidgetContext();
   return (
     <Button
       appearance={{
         theme: isObjection ? 'danger' : 'primary',
         size: 'medium',
       }}
       type="submit"
-      disabled={!canBeStaked}
```

Comment:
`canBeStaked` is an inflexible abstraction that means different things for different parts of the staking widget, so have decomposed to its constituent booleans